lib/experimental/fonts/bc-layer/class-gutenberg-fonts-api-bc-layer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Fonts API BC Layer helpers.
+ *
+ * BACKPORT NOTE: Do not backport this file to Core.
+ * This file is now part of the API's Backwards-Compatibility (BC) layer.
+ *
+ * @package    Gutenberg
+ * @subpackage Fonts API
+ * @since      15.7.0
+ */
+
+/**
+ * Class Gutenberg_Fonts_API_BC_Layer
+ *
+ * BACKPORT NOTE: Do not backport this file to Core.
+ *
+ * @since X.X.X
+ * @deprecated 16.3.0 Fonts API is not supported.
+ */
+class Gutenberg_Fonts_API_BC_Layer {
```

Comment:
Personal preference and definitely not a blocker, but couldn't we be more verbose?
`BC` can mean a lot of things, and `Backwards_Compatibility` may be a bit longer but is definitely easier to understand

Reply:
You're right @aristath that spelling out `Backwards_Compatibility` is more understandable. But it's too late to make that change.

Why is it too late to change?
This class already shipped in Gutenberg as part of the Fonts API. In this PR, the class name is copied here just in case an extender is using it.